src/core/subdirdata.cpp

```diff
@@ -106,65 +108,101 @@ namespace mamba
-                {
-                    max_age = Context::instance().local_repodata_ttl;
-                }
-                else if (Context::instance().local_repodata_ttl == 1)
-                {
-                    // TODO error handling if _cache_control key does not exist!
-                    auto el = m_mod_etag.value("_cache_control", std::string(""));
-                    max_age = get_cache_control_max_age(el);
-                }
+                LOG_INFO << "Found cache at '" << json_file.string() << "'";
+                m_mod_etag = read_mod_and_etag(json_file);
 
-                auto cache_age_seconds
-                    = std::chrono::duration_cast<std::chrono::seconds>(cache_age).count();
-                if ((max_age > cache_age_seconds || Context::instance().offline))
+                if (m_mod_etag.size() != 0)
                 {
-                    // cache valid!
-                    LOG_INFO << "Using cache " << m_repodata_url
-                             << " age in seconds: " << cache_age_seconds << " / " << max_age;
-                    std::string prefix = m_name;
-                    prefix.resize(PREFIX_LENGTH - 1, ' ');
-                    Console::stream() << prefix << " Using cache";
-
-                    m_loaded = true;
-                    m_json_cache_valid = true;
-
-                    // check solv cache
-                    auto solv_age = check_cache(m_solv_fn, now);
-                    LOG_INFO << "Solv cache age in seconds: "
-                             << std::chrono::duration_cast<std::chrono::seconds>(solv_age).count();
-                    if (solv_age != fs::file_time_type::duration::max()
-                        && solv_age.count() <= cache_age.count())
+                    int max_age = 0;
+                    if (Context::instance().local_repodata_ttl > 1)
+                    {
+                        max_age = Context::instance().local_repodata_ttl;
+                    }
+                    else if (Context::instance().local_repodata_ttl == 1)
                     {
-                        LOG_INFO << "Also using .solv cache file";
-                        m_solv_cache_valid = true;
+                        // TODO error handling if _cache_control key does not exist!
+                        auto el = m_mod_etag.value("_cache_control", std::string(""));
+                        max_age = get_cache_control_max_age(el);
+                    }
+
+                    auto cache_age_seconds
+                        = std::chrono::duration_cast<std::chrono::seconds>(cache_age).count();
+                    if ((max_age > cache_age_seconds || Context::instance().offline))
+                    {
+                        // valid json cache found
+                        if (!m_loaded)
+                        {
+                            LOG_DEBUG << "Using JSON cache";
+                            LOG_TRACE << "Cache age: " << cache_age_seconds << "/" << max_age
```

Comment:
this should be log info.

Reply:
a log info should be limited to the actions the user asked for, not internal machinery
I posted [here](https://github.com/mamba-org/mamba/issues/996#issuecomment-937637727) what I would like to use as a log policy

Reply:
Imaging being an sysadmin trying to debug why mamba won't load cached files though ... those are the users of our logs (and we are, when people are filing bugs). Making these logs to TRACE gives us little chance to figure out whats going on.